site/docs/components/pagination.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Pagination
+navTitle: Pagination (not built)
+summaryParagraph: Pagination separates large bodies of content into separate pages. You can access each page via a shared index of links.
+tags: ["Pages"]
+needToKnow:
+  - Perform usability testing to decide where to split content into pages.
+  - This has been designed but not built.
+---
+
+import DosAndDonts from "docs-components/DosAndDonts"
+import Do from "docs-components/Do"
+import Dont from "docs-components/Dont"
+
+## To keep in mind
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D4332%253A26712" allowfullscreen></iframe>
+
+- Place Pagination at the bottom of the long list of content that has been split up into pages.
+- Let people navigate to the Previous and Next set of items in the paged list.
+- Show when people are on the first or the last page by disabling the its pagination link.
+- Where possible, load each set of items without reloading the whole page.
+- Highlight the Current page as well as available pages in Pagination, such as the total number of results.
+- To decide where to split content into pages, consider:
+    - Page and data load time
+    - Amount of content in each entry
+    - Screen real estate available
+    - How many items a person may need to compare at once
+- Compare [Pagination and Infinite Scroll](/guidelines/pagination-infinite-scroll) with a "View more" button.
+
+## When to use and when not to use
+
+<DosAndDonts>
+<Do>
+
+- Use Pagination when people are unlikely to need to see **all** the content.
+- Use Pagination when the first page is enough to make a decision.
+- Use Pagination when loading all the content at once would not be possible technically or not be performant. If it takes more than 10 seconds to load all the content, paginate.
+- Use Pagination when keeping your place is important or you want to bookmark a specific page.
+- Use Pagination when there's lot of screen real estate.
+- Consider using pagination when a list contains more than 50 items of a single line of text each.
+
+</Do>
+<Dont>
+
+- Avoid pagination when seeing every item and comparing them at the same time is important to the task.
+- Manual pagination may be less efficient than [Infinite Scroll](/components/infinite-scroll) because it requires additional user interaction to click or tap a link to navigate to the next page as well as cognitive load to choose a page number or pagination button.
```

Comment:
_Somewhat_ out of date I guess because both require a click/tap, but the part about cognitive load stands.